aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/OutboxMutationFailedEvent.java

```diff
@@ -51,6 +51,31 @@ private OutboxMutationFailedEvent(
         this.model = model;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null || getClass() != obj.getClass()) {
+            return false;
+        }
+        OutboxMutationFailedEvent<?> that = (OutboxMutationFailedEvent<?>) obj;
+        return getErrorType() == that.getErrorType() &&
+            getOperation() == that.getOperation() &&
+            getModelName().equals(that.getModelName()) &&
+            getModel().equals(that.getModel());
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(
```

Comment:
[`Objects.hash(..)`](https://developer.android.com/reference/java/util/Objects#hash(java.lang.Object...)) is only supported from API level 19. This should say [`ObjectsCompat.hash(...)`](https://developer.android.com/reference/androidx/core/util/ObjectsCompat#hash(java.lang.Object...)).

I thought maybe it could be desugared in the Android 4.x tools, but that doesn't look to be the case, either.